src/json/json_parser.h

```diff
@@ -0,0 +1,137 @@
+/*******************************************************************************
+*   (c) 2018, 2019 Zondax GmbH
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+********************************************************************************/
+
+#pragma once
+
+#include <jsmn.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <string.h>
+#include "common/parser_common.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if defined(LEDGER_SPECIFIC)
```

Comment:
Why keeping such define condition? Do we have some cases where the flag is not defined?

Reply:
I don't know, this is zondax stuff, I'd prefer not touching it...